dracut/30coreos-installer/coreos-installer.service

```diff
@@ -1,15 +1,9 @@
 [Unit]
 Description=CoreOS Installer
-Requires=network-online.target
-After=network-online.target
```

Comment:
I think we dropped this? Was that intentional?

Reply:
AFAIK it doesn't serve any purpose since we rely on dracut to set up networking, which doesn't integrate with the special `network*` targets.

Reply:
:cry: Can we add some comments explaining that this needs networking? Otherwise we might get bitten when we switch to NM in the initramfs.